src/Files.App/Data/Enums/ContextFlyoutItemType.cs

```diff
@@ -6,12 +6,12 @@ namespace Files.App.Data.Enums
 	/// <summary>
 	/// Defines constants that specify item type of ContextMenuFlyout on Windows.
 	/// </summary>
-	public enum ContextMenuFlyoutItemType
+	public enum ContextFlyoutItemType
 	{
 		/// <summary>
 		/// Button type.
 		/// </summary>
-		Item,
```

Comment:
I think it's usually refered to as "menu item" as opposed to "button".

Reply:
Can you rename back to `Item`?

Reply:
Done